x/genaccounts/legacy/v0_36/migrate.go

```diff
@@ -0,0 +1,138 @@
+// DONTCOVER
+// nolint
+package v0_36
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	v034distr "github.com/cosmos/cosmos-sdk/x/distribution/legacy/v0_34"
+	v034accounts "github.com/cosmos/cosmos-sdk/x/genaccounts/legacy/v0_34"
+	v034staking "github.com/cosmos/cosmos-sdk/x/staking/legacy/v0_34"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+const (
+	notBondedPoolName = "NotBondedTokensPool"
+	bondedPoolName    = "BondedTokensPool"
+	feeCollectorName  = "FeeCollector"
+	mintModuleName    = "mint"
+)
+
+// Migrate accepts exported genesis state from v0.34 and migrates it to v0.36
```

Comment:
Looks like copy-pasta. Remove the last sentence. Feel free to replace it with what is actually being done here 👍